docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+bar
```

Comment:
I'm still not sure how to put it, the original bar is written all-lowercase (like unix commands use to) but it's hard to tell from the noun then.